src/utils/get-pile-prop.js

```diff
@@ -0,0 +1,6 @@
+import { isFunction } from '@tensorflow/tfjs-core/dist/util';
```

Comment:
```suggestion
import { isFunction } from '@flekschas/utils';
```
No new dependencies unless they are really necessary.